Library/Formula/libbpg.rb

```diff
@@ -3,6 +3,7 @@ class Libbpg < Formula
   homepage "http://bellard.org/bpg/"
   url "http://bellard.org/bpg/libbpg-0.9.6.tar.gz"
   sha256 "2800777d88a77fd64a4a9036b131f021a5bda8304e1dbe7996dd466567fb484e"
+  revision 1
```

Comment:
I don't really know, I thought that if formula is updated without version update, `revision` was needed.
I can remove it, but I think it can warn users that default encoder change from `jctvc` to `x265`, this is one of major change between `0.9.5` and `0.9.6`